spec/models/chargeback_container_project_spec.rb

```diff
@@ -123,6 +123,21 @@
       expect(subject.net_io_used_metric).to eq(@net_usage_rate * @metric_size)
       expect(subject.net_io_used_cost).to eq(@net_usage_rate * @hourly_rate * @metric_size)
     end
+
+    it "fixed_compute" do
+      cbrd = FactoryGirl.build(:chargeback_rate_detail_fixed_compute_cost,
+                               :chargeback_rate_id => @cbr.id,
+                               :per_time           => "hourly")
+      cbt = FactoryGirl.create(:chargeback_tier,
+                               :chargeback_rate_detail_id => cbrd.id,
+                               :start                     => 0,
+                               :finish                    => Float::INFINITY,
+                               :fixed_rate                => 0.0,
+                               :variable_rate             => @hourly_rate.to_s)
+      cbrd.chargeback_tiers = [cbt]
```

Comment:
maybe it is posible to to add it as parameter in create/build method  
```ruby
FactoryGirl.build(::chargeback_rate_detail_fixed_compute_cost,
                             .... 
                             :chargeback_tiers => [cbt])
```
then dont need call `save` as you have later